packages/fxa-auth-server/scripts/delete-unverified-accounts/delete-unverified-accounts.ts

```diff
@@ -0,0 +1,308 @@
+    );
+  }
+  if ((hasUid || hasEmail) && hasDateRange) {
+    throw new Error(
+      'Sorry, but the script does not support uid/email arguments and a date range in the same invocation.'
+    );
+  }
+  if (limit <= 0) {
+    throw new Error('The limit should be a positive integer.');
+  }
+
+  const useSpecifiedAccounts = hasUid || hasEmail;
+  const useDateRange = !useSpecifiedAccounts && hasDateRange;
+
+  if (isDryRun) {
+    console.log(
+      'Dry run mode is on.  It is the default; use --dry-run=false when you are ready.'
+    );
+
+    return dryRun(program, useSpecifiedAccounts, useDateRange, limit);
+  }
+
+  const config = appConfig.getProperties();
+  const log = initLog({
+    ...config.log,
+  });
+  const statsd = new StatsD({ ...config.statsd });
+  const redis = initRedis(
+    { ...config.redis, ...config.redis.sessionTokens },
+    log
+  );
+  const db = DB(
+    config,
+    log,
+    Token(log, config),
+    random.base32(config.signinUnblock.codeLength) as any // TS type inference is failing pretty hard with this
+  );
+  const fxaDb = await db.connect(config, redis);
+
+  Container.set(AppConfig, config);
+  Container.set(AuthLogger, log);
+
+  const authFirestore = setupFirestore(config);
+  Container.set(AuthFirestore, authFirestore);
+  const currencyHelper = new CurrencyHelper(config);
+  Container.set(CurrencyHelper, currencyHelper);
+  const stripeHelper = createStripeHelper(log, config, statsd);
+  Container.set(StripeHelper, stripeHelper);
+
+  const accountTasks = AccountTasksFactory(config, statsd);
+
+  if (useSpecifiedAccounts) {
+    const { uids, emails } = limitSpecifiedAccounts(program, limit);
+
+    for (const x of uids) {
+      const acct = await fxaDb.account(x);
+      if (acct && acct.emailVerified) {
+        console.error(`Account with uid ${x} is verified.  Skipping.`);
+        continue;
+      }
+      const result = await accountTasks.deleteAccount({
+        uid: x,
+        customerId: (await getAccountCustomerByUid(x))?.stripeCustomerId,
+        reason,
+      });
+      console.log(`Created cloud task ${result} for uid ${x}`);
+    }
+
+    for (const x of emails) {
+      const acct = await fxaDb.accountRecord(x);
+      if (acct && acct.emailVerified) {
+        console.error(`Account with email ${x} is verified.  Skipping.`);
+        continue;
+      }
+      const result = await accountTasks.deleteAccount({
+        uid: acct.uid,
+        customerId: (await getAccountCustomerByUid(acct.uid))?.stripeCustomerId,
+        reason,
+      });
+      console.log(`Created cloud task ${result} for uid ${x}`);
+    }
+  }
+
+  if (useDateRange) {
+    if (program.tableScan !== 'true') {
+      console.log('Please call with --table-scan if you are sure.');
+      return 0;
+    }
+
+    await deleteAccountInDateRange(program.startDate, program.endDate, {
+      taskLimit,
+      reason,
+      accountTasks,
+      auth: config.auth,
+    });
+  }
+
+  return 0;
```

Comment:
~~Looks like a few code paths here return `0`, but I'm not certain that the value is being used.~~

Well _actually_&hellip; I think the `export default init;` below on L308 uses this as the `exitCode` above on `packages/fxa-auth-server/scripts/delete-unverified-accounts.ts#L13`. 🤔